src/binaries/trezord-go/bin/download.sh

```diff
@@ -5,3 +5,4 @@ set -e
 
 # Bridge needs an older glibc so we are pinning to nixos-21.05 (stable) as of 2021-07-02
 nix-shell -p autoPatchelfHook -I "nixpkgs=https://github.com/NixOS/nixpkgs/archive/e9148dc1c30e02aae80cc52f68ceb37b772066f3.tar.gz" --run "autoPatchelf trezord-*"
+nix-shell -p autoPatchelfHook  --run "patchelf --set-interpreter /nix/store/0y64vbl8hc0xqq5ag6wwj5r3pr0l6z2s-glibc-2.33-56/lib/ld-linux-aarch64.so.1 ./src/binaries/trezord-go/bin/*-arm64"
```

Comment:
Hardcoding the path is definitely wrong. We can't autopatch it? @mmilata might help here.

Reply:
I think if we change it to `nix-shell -p autoPatchelfHook  --run "patchelf --set-interpreter /lib/ld-linux-aarch64.so.1 ./src/binaries/trezord-go/bin/*-arm64"` it should work and use the current store path

Reply:
Ok so we can't, it's not working without setting it like that.